tests/integration/components/feed-item-test.js

```diff
@@ -9,7 +10,43 @@ test('it renders', function(assert) {
   // Set any properties with this.set('myProperty', 'value');
   // Handle any actions with this.on('myAction', function(val) { ... });
 
-  this.render(hbs`{{feed-item}}`);
+  let article = {
+    author: {
+      image: 'https://static.productionready.io/images/smiley-cyrus.jpg',
+      username: 'Alon Bukai'
+    },
+    createdAt: moment('1995-12-25'),
+    favoritesCount: 9000,
+    body: 'This is feed item content',
+    slug: 'feed-item-title',
+    title: 'Feed Item Title'
+  };
 
-  assert.equal(this.$().text().trim(), '');
+  this.set('article', article);
+  this.render(
```

Comment:
There's a standing issue that I filed against Prettier because of the way it mangles these inline `hbs` tags.

https://github.com/prettier/prettier/issues/1014

I think they're working on deciding what the right fix is.